src/plugins/virtualbox/vm.cpp

```diff
@@ -0,0 +1,121 @@
+// albert - a simple application launcher for linux
```

Comment:
Kinda, sorta, ... This class prevents the `Extension` class from needing to coordinate 2 separate arrays (or vectors or QLists whatever) like names_ and uuids_. This class does represent a VM. It holds its name and uuid. And it can produce an VMItem of the VM it represents. It is more a container class to get name and uuid into a single object. And further I moved the parsing code into the constructor...